src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let output;
+  if (operation === '+') {
+    output = Big(numberOne).plus(Big(numberTwo));
+  }
+  if (operation === '-') {
+    output = Big(numberOne).minus(Big(numberTwo));
+  }
+  if (operation === '÷') {
+    output = Big(numberOne).div(Big(numberTwo));
+  }
+  if (operation === '×') {
+    output = Big(numberOne).mul(Big(numberTwo));
+  }
+  return output;
+}
```

Comment:
- [ ] operate should switch operation and use plus, minus, times, div and modifier(
turning the number into a percentage based number, `e.g. 81 % == .81`)